feat(platform logs): add filter for admin

phiz71 opened the pull request:
## Issue

https://gravitee.atlassian.net/browse/APIM-4079
https://github.com/gravitee-io/issues/issues/9599

## Description

Add a filter for APIs and applications when current user is admin, to force looking into one environment only.


<!-- Storybook placeholder -->
---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-chkxydddea.chromatic.com)
<!-- Storybook placeholder end -->

jhaeyaert commented on gravitee-apim-rest-api/gravitee-apim-rest-api-standalone/gravitee-apim-rest-api-standalone-distribution/src/main/resources/config/gravitee.yml:
```diff
@@ -230,6 +230,10 @@ services:
+#  logs:
+     # Maximum number of APIs and applications a query to the analytics database can contain when searching for platform logs.
+     # Default value of 250, means the query can contain at most 250 apis and 250 applications.
+#    maxItemsNumberInPlatformLogQuery: 250
```
As discussed, the `max_clause_count` seems to be a limitation applied per filter, meaning that you can have a `apis` filter with up to 1024 and an `applications` filter with up to 1024. Maybe we should change a bit the current implementation?
Regarding the default value of 250, elastic sets it to 1024 by default. Should we align it?

phiz71 replied:
I've decided to remove all the limitation part from this PR.
We already had the same kind of issue for analytics: https://gravitee.atlassian.net/browse/APIM-1686
And potential maxClauseCount issue has not been taken into account.

On top of that, limit the number of API and applications could be seen as a big regression for users and I finally don't think that it is the kind of decision we can make at support while fixing a bug.

jhaeyaert replied:
Agree

phiz71 committed:
feat(platform logs): add filter for admin

Add a filter for APIs and applications when current user is admin, to force looking into one environment only.

https://gravitee.atlassian.net/browse/APIM-4079

phiz71 committed:
fix: use the same filter for analytics as for log
